src/lancode.c

```diff
@@ -298,7 +297,7 @@ int send_lan_message(int opcode, char *message) {
 	lan_send(sendbuffer);
     }
     if (opcode == FREQMSG) {
-	sendbuffer[10] = '\0';
+	strcat(sendbuffer, "\n");
```

Comment:
Is new line needed?

Reply:
send_freq() formats the QRG as xxxx.y with one decimal. Without the newline the decimal gets dropped.

See also above.